agent/server.go

```diff
@@ -72,6 +74,13 @@ func (r *server) LocalStatus(ctx context.Context, req *pb.LocalStatusRequest) (r
 	return resp, nil
 }
 
+// Time sends back the target node server time
+func (r *server) Time(ctx context.Context, req *pb.TimeRequest) (*pb.TimeResponse, error) {
+	return &pb.TimeResponse{
+		Timestamp: pb.NewTimeToProto(time.Now()),
```

Comment:
Does not look like this is returning UTC as expected by the checker.

Reply:
In Go timestamp always includes timezone information. But I added "UTC()" anyway.